lib/wallet/walletdb.js

```diff
@@ -194,6 +195,9 @@ class WalletDB extends EventEmitter {
       this.state.height,
       this.state.startHeight);
 
+    if (this.getWalletOptions)
+      await this.ensure(this.getWalletOptions());
```

Comment:
This call a callback passed down from the wrapper to get the restored wallet configuration.

It could probably just be a static object that we populate before instantiating the `FullNode`, this was just the most flexible.